### PR TITLE
Fixes #36160 - add display_name field to Host GraphQL type

### PR DESCRIPTION
**Pull request by nofaralfasi**

### CODE REVIEW

**Review by sbernhard — DISMISSED** [review]
LGTM. Thanks.

**Comment by adamruzicka** [comment]
[test Redmine issues]

**Review by adamruzicka — DISMISSED** [review]
LGTM

Could you please rebase? Maybe it would make the redmine issue check turn green

File: app/graphql/types/host.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ class Host < BaseObject
     global_id_field :id
     timestamps
     field :name, String
+    field :display_name, String, method: :to_label
```
Comment (kamils-iRonin) [inline app/graphql/types/host.rb:10]: I suggest adding this field to the host query test https://github.com/theforeman/foreman/blob/5423d2c46a89f7f66b9d89a13f5fe4d5b481cab5/test/graphql/queries/host_query_test.rb#L3-L4

**Comment by stejskalleos** [comment]
Unit tests are failing due to the changes in the PR: `Queries::HostQueryTest:`

**Comment by stejskalleos** [comment]
[test katello]

**Review by stejskalleos — APPROVED** [review]
:green_apple: **LGTM**

**Comment by stejskalleos** [comment]
Thanks @nofaralfasi and others!

**Pull request merged**